examples/src/main/scala/za/co/absa/atum/examples/SampleMeasurements3.scala

```diff
@@ -56,6 +57,15 @@ object SampleMeasurements3 extends Eventually {
         throw new Exception("_INFO file not found at data/output/stage1_job_results")
     }
 
+    val jsonInfoFile = FileUtils.readFileToString("data/output/stage1_job_results/_INFO")
+    val measureObject1: ControlMeasure = SerializationUtils.fromJson[ControlMeasure](jsonInfoFile)
+    val checkpoint = measureObject1.checkpoints.filter(_.name == "checkpoint1").head
+
+    if (checkpoint.software.isEmpty || checkpoint.version.isEmpty ||
+      !checkpoint.software.contains("MySoftware") || !checkpoint.version.contains("v007")) {
```

Comment:
```suggestion
    if (!checkpoint.software.contains("MySoftware") || !checkpoint.version.contains("v007")) {
```

`.contains` will cover the empty case, too.